src/Tinkoff.ISA.API/Program.cs

```diff
@@ -1,16 +1,17 @@
-﻿using Microsoft.AspNetCore;
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
 
 namespace Tinkoff.ISA.API
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static Task Main(string[] args)
         {
-            CreateWebHostBuilder(args).UseUrls("http://*:5000").Build().Run();
+            return CreateWebHostBuilder(args).UseUrls("http://*:5000").Build().RunAsync();
```

Comment:
what is the reason to change it to RunAsync? Is there any other processes?
Is there any difference in runtime level?